lib/libutils/isoc/arch/arm/setjmp_a64.S

```diff
@@ -51,6 +51,12 @@ setjmp:
 #define REG_ONE(REG1, OFFS)		str REG1, [x0, OFFS]
 	GPR_LAYOUT
 	FPR_LAYOUT
+#ifdef CFG_TA_FTRACE_SUPPORT
+	stp	x29, x30, [sp, #-16]!
+	mov	x29, sp
```

Comment:
Why are you storing `sp` in `x29`?

Reply:
Its normal flow for C function calls to create stack frames. As we are making function calls like `C->assembly->C`, so I kept the same pattern.

Reply:
OK